requirements.txt

```diff
@@ -1,19 +1,19 @@
 boto==2.45.0
 cssselect==1.0.1
 cssutils==1.0.1
-dj-database-url==0.4.2
 Django==1.10
 django-appconf==1.0.1
 django-compressor==2.1
 django-ses-backend==0.1.1
 djmail==0.11.0
+funcsigs==1.0.2
 futures==3.0.3
-gunicorn==19.6.0
 lxml==3.7.2
+mock==2.0.0
+pbr==1.10.0
 premailer==2.9.6
+psycopg2==2.6.1
 rcssmin==1.0.6
 requests==2.9.1
 rjsmin==1.0.12
 six==1.10.0
-whitenoise==3.2.3
-psycopg2==2.6.1
```

Comment:
@andela-mallan Seems like some requirements here are being replaced. Can you update your requirements file with the ones highlighted in red then push the changes?